### PR TITLE
Slotmachines

### PR DESCRIPTION
**Pull request by Dawnshock**
## About The Pull Request

Adds slotmachines to the bar

## Why It's Good For The Game

Another thing to waste your cash

## Changelog
add: Added slotmachines

### CODE REVIEW

**Comment by SpadesSlick** [comment]
Please use map merger for map edits.

File: code/game/machinery/slotmachine.dm [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+	anchored = 1.0
+	var/spinning = 0
+	var/bet = 0
+	var/jackpot = 0
+	var/plays = 0
+	var/slots = list()
+	var/icon_type
+	use_power = 1
+	idle_power_usage = 10
+	//var/list/fruits = list("Cherry","Apple","Blueberry","Bell","Watermelon","JACKPOT")
```
Comment (mattroks101) [inline code/game/machinery/slotmachine.dm:16]: What's this for?

**Comment by mattroks101** [comment]
Is this dead in the water?

**Comment by mattroks101** [comment]
Feel free to open a new PR when you plan on finishing this.

**Pull request closed without merging**